### PR TITLE
[ChatAI] Create thread 

### PR DESCRIPTION
**Pull request by jiyoung-an**
- add create thread method and test code.
- setup assistant for thread test.

### CODE REVIEW

**Review by platfowner — APPROVED** [review]
LGTM
Thanks for the hard work!

File: test/ai/threads/threads.test.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+const objectId = '0x45E89F37Cee508cf0D4F6e74b35EfeBdd90BD731';
+const appId = 'ainft721_0x45e89f37cee508cf0d4f6e74b35efebdd90bd731';
+const aiName = 'ainize_test14';
+const tokenId = '2';
+const address = '0x7ed9c30C9F3A31Daa9614b90B4a710f61Bd585c0';
+const threadId = 'thread_000000000000000000000001';
+
+describe('Thread', () => {
+  beforeAll(async () => {
```
Comment (platfowner) [inline test/ai/threads/threads.test.ts:17]: 👍

File: src/types.ts [hunk 2]
```diff
@@ -1194,17 +1209,39 @@ export interface OpenAIAssistantDeleteParams extends AssistantDeleteParamsBase {
...
+export interface ThreadMessage {
   id: string;
-  deleted: boolean;
+  content: string;
+  role: 'user' | 'assistant';
+  metadata: object | null;
+  created_at: number;
+}
+
+interface ThreadCreateParamsBase {
```
Comment (jiyoung-an) [inline src/types.ts:1229]: when creating thread and message, metadata is inputed and sent only to OpenAI,
and is not save on the blockchain.

File: src/types.ts [hunk 3]
```diff
@@ -1194,17 +1209,39 @@ export interface OpenAIAssistantDeleteParams extends AssistantDeleteParamsBase {
...
+  tokenId: string;
+  messages?: Array<ThreadCreateParams.Message>;
+  metadata?: object | null;
```
Comment (jiyoung-an) [inline src/types.ts:1232]: when creating thread and message, metadata is inputed and sent only to OpenAI, and is not save on the blockchain.

**Comment by jiyoung-an** [comment]
Thank you for reviews and advices! @platfowner now merging...

**Pull request merged**